### ISSUE TITLE
require-input-label false positive 

### ISSUE THREAD

**Issue by amk221** [ember-template-lint/ember-template-lint#3006]
```javascript
const input = 'foo';

await render(<template>
  <MyComponent @string={{input}} />
</template>);
```

...it's picking up `{{input}}`

**Comment by bmish** [comment]
CC: @MelSumner who implemented the rule.

**Comment by MelSumner** [comment]
I can understand why it's picking it up, but I don't know how to edit the rule without making it less effective. Can you rename `{{input}}` to be something more precise for your use case?

**Comment by lifeart** [comment]
Closing issue because issue ma be solved with different path name as @MelSumner suggested
